src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -132,4 +144,61 @@ public CommandBase turnDegrees(double degrees, double speed) {
         cmd.setName("Turn Degrees");
         return cmd;
     }
+
+    public CommandBase slowTurn(boolean isTurningRight) {
+        CommandBase cmd = new RunCommand(() -> {
+            if (isTurningRight == true) {
```

Comment:
Just use `if(isTurningRight) {}`. Using `== true` is redundant. Also below you can use the `else` without `else if`

Reply:
resolved